source/reference/command/serverStatus.txt

```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
 
    .. versionadded:: 5.0
 
+.. serverstatus:: shardingStatistics.numHostsTargeted
+
+   Indicates the number of shards targeted for ``CRUD`` operations and 
+   aggregation commands. When a ``CRUD`` operation or aggregation
+   command is run, the following metrics will be incremented.
+
+   .. include:: /includes/fact-server-status-num-hosts-targeted.rst
+
+   .. note:: Running the :dbcommand:`serverStatus` on :binary:`mongos` 
+            will provide insight into the aggregation operations find, 
+            insert, update, delete and aggregate that run on a sharded 
+            cluster. 
+            
+            Multi-shard operations can either be 
+            :ref:`scatter-gather<sharding-query-patterns>`
+            or shard specific. Multi-shard scatter-gather operations can
+            :ref:`consume more resources<sharding-troubleshooting-scatter-gather>`,
+            by using the 
+            :serverstatus:`shardingStatistics.numHostsTargeted`
+            metrics you can tune the aggregation queries that run on a 
+            sharded cluster.
```

Comment:
There's some weird indentation happening here. We should be doing:

```
.. note::

   Running the :dbcommand:`serverStatus` command on...
   
```

Reply:
Updated.